### PR TITLE
Edit place functionality

### PR DESCRIPTION
**Pull request by RoxVer**
Classes for feedback functionality added and not necessary edit-place files deleted

### CODE REVIEW

File: src/main/webapp/resources/app/components/place/edit.place.component.js [hunk 1]
```diff
@@ -1,50 +0,0 @@
-'use strict';
-
-angular.module('greenApp')
-.controller('EditPlaceController')
```
Comment (47souls) [inline src/main/webapp/resources/app/components/place/edit.place.component.js:4]: Could you please explain why have you deleted edit.place.template.html and place/edit.place.component.js. If it was mistake, then add it again please.

Comment (RoxVer) [reply src/main/webapp/resources/app/components/place/edit.place.component.js:4]: Misha said to add edit place functionality to one component - place component, so it's not a mistake.

Comment (47souls) [reply src/main/webapp/resources/app/components/place/edit.place.component.js:4]: So now we don't have this functionality at all? Or you have removed this piece of code somewhere else?)

Comment (RoxVer) [reply src/main/webapp/resources/app/components/place/edit.place.component.js:4]: I'm gonna place it to place component

Comment (47souls) [reply src/main/webapp/resources/app/components/place/edit.place.component.js:4]: Ok, thanks for explaination

Comment (RoxVer) [reply src/main/webapp/resources/app/components/place/edit.place.component.js:4]: These docs were created by me, so no worries)

**Comment by ghost** [comment]
reviewed

**Pull request merged**